Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -312,4 +318,56 @@ extension BrowserViewController: ReaderModeDelegate {
             toolbar.updateReaderModeState(state)
         }
     }
+}
+
+extension BrowserViewController: BrowserLongPressGestureDelegate {
+    func browser(browser: Browser, elementsDidLongPressed elements: [ElementType]) {
+        var actionSheetController = UIAlertController(title: "", message: "Mozilla Firefox", preferredStyle: UIAlertControllerStyle.ActionSheet)
```

Comment:
Please pass nil as the title instead of an empty string. Also, go ahead and pass nil for message too -- we don't need to show the browser name in this dialog.